charts/incubator/minecraft-bungeecord/docs/installation_notes.md

```diff
@@ -1,13 +1,6 @@
 # Installation Notes
 
-- Set `imageSelector` to any of values from the list default is `image`. ex: `image` (java17), `j11Image`(java11), `j8Image`(java8).
-
-- Set `ENABLE_RCON` to **true** in order to show `RCON_PASSWORD` and `RCON_PORT`.
-
+- Set `Select Image` to one of the values from the list. `Java 17`, `Java 11`, `Java 8`. Default is `Java 17`.
 - Set `RCON_PASSWORD` to a SuperSecurePassword.
-
-- Set `RCON_PORT` to **25577**.
-
```

Comment:
Uses can change port on the service section. It's recommended to NOT change internal ports.
Unless there is a reason for that